classes/ActionScheduler_LogEntry.php

```diff
@@ -4,12 +4,31 @@
  * Class ActionScheduler_LogEntry
  */
 class ActionScheduler_LogEntry {
-	protected $action_id = '';
-	protected $message = '';
+	protected $action_id =  '';
+	protected $message =  '';
+	protected $date;
 
-	public function __construct( $action_id, $message ) {
+	/**
+	 * Constructor
+	 *
+	 * @param mixed  $action_id	Action ID
```

Comment:
There should be a space instead of a tab between `$action_id` and `Action ID`. Also, is there a reason the type is "mixed" instead of "int"?

Reply:
> is there a reason the type is "mixed" instead of "int"?

I assume it's because there is no guarantee an ID will always be an `int`. It is an `int` with built-in datastores, but it could just as easily be a string if a hash or SHA was used to refer to an action by other data stores.

Reply:
Ah-hah, makes perfect sense. Thanks 🙂